src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -51,33 +51,33 @@ _SUPPORTING_FILE_ATTRIBUTES = [
     'xibs',
 ]
 
-# Set of rules with implicit <label>.ipa IPA outputs.
+# List of rules with implicit <label>.ipa IPA outputs.
 # TODO(b/33050780): This is only used for the native rules and will be removed
 # in the future
-_IPA_GENERATING_RULES = set([
+_IPA_GENERATING_RULES = [
```

Comment:
Any reason not to use depset here? These sets are mostly used for "contains" checks, which I assume are faster in depsets?